### PR TITLE
[BX-1052] Fix explainer keyboard nav

### PR DESCRIPTION
**Pull request by BrodyHughes**
- added `data-is-explainer-sheet` to the `ExplainerSheet`s
- added `isExplainerSheet` to the `simulateTab` so it prioritizes `ExplainerSheets`
- This makes explainers navigatable with keyboard.

POW:
<video src="https://github.com/rainbow-me/browser-extension/assets/41711440/811c02b8-56c9-4218-9832-d9179fbcd8eb" controls="controls" style="max-width: 730px;"></video>

### CODE REVIEW

**Comment by brunobar79** [comment]
@BrodyHughes  can you add PoW?

File: src/entries/popup/utils/simulateTab.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 export const simulateTab = (forwards: boolean) => {
   const activeElement = document.activeElement;
   if (activeElement) {
     const modal = getActiveModal();
-    const target = modal || document;
+    const explainer = getExplainerSheet();
+    const target = explainer || modal || document;
+    console.log('modal:', modal);
+    console.log('explainer:', explainer);
+    console.log('target:', target);
```
Comment (derHowie) [inline src/entries/popup/utils/simulateTab.ts:11]: 👀

File: src/entries/popup/components/ExplainerSheet/ExplainerSheet.tsx [hunk 2]
```diff
@@ -118,14 +118,13 @@ export const ExplainerSheet = ({
...
   }, []);
-
   return (
     <BottomSheet
       onClickOutside={onClickOutside || actionButton?.action}
       zIndex={zIndexes.EXPLAINER_BOTTOM_SHEET}
       show={show}
     >
-      <Box testId={`explainer-sheet-${testId}`}>
+      <Box testId={`explainer-sheet-${testId}`} isExplainerSheet>
```
Comment (greg-schrammel) [inline src/entries/popup/components/ExplainerSheet/ExplainerSheet.tsx:127]: isn't an explainer sheet a modal? is this new prop different than the `isModal` one?

Comment (BrodyHughes) [reply src/entries/popup/components/ExplainerSheet/ExplainerSheet.tsx:127]: It is, but I think this is necessary in the case that there are nested BottomSheets (like ReviewSheet > ExplainerSheet)

**Review by derHowie — APPROVED** [review]

**Review by greg-schrammel — APPROVED** [review]

**Pull request merged**